src/inference/gibbs.jl

```diff
@@ -83,11 +83,24 @@ metadata(t::GibbsTransition) = (lp = t.lp,)
 DynamicPPL.getlogp(t::GibbsTransition) = t.lp
 
 # extract varinfo object from state
-getvarinfo(state) = state.vi
-getvarinfo(state::AbstractVarInfo) = state
+"""
+    gibbs_varinfo(model, sampler, state)
+
+Return the variables corresponding to the current `state` of the Gibbs component `sampler`.
```

Comment:
What is meant by "variables" in the above? Is it supposed to be `VarInfo`?

Reply:
Yes, maybe that should be made more explicit.